store/src/java/com/zimbra/cs/service/UserServlet.java

```diff
@@ -278,7 +279,11 @@ private void sendError(UserServletContext ctxt, HttpServletRequest req, HttpServ
             && (req.getMethod().equalsIgnoreCase("POST") || req.getMethod().equalsIgnoreCase("PUT"))) {
             resp.sendError(HttpServletResponse.SC_UNAUTHORIZED, L10nUtil.getMessage(MsgKey.errMustAuthenticate, req));
         } else {
-            resp.sendError(HttpServletResponse.SC_NOT_FOUND, message);
+            if (ctxt.getAuthAccount() instanceof GuestAccount && ctxt.basicAuthAllowed() ) {
```

Comment:
The above code hints that `ctxt` can be null under some circumstances.

Reply:
Fixed